storage/flush.go

```diff
@@ -101,17 +70,24 @@ func (m *flushManager) Flush(curr time.Time) error {
 	m.flushInProgress = true
 	m.Unlock()
 
+	// create flush-er
+	flush, err := m.pm.StartFlush()
+	if err != nil {
+		return err
+	}
+	defer flush.Done()
```

Comment:
Does `flush.Done()` return an error?  If so should it be checked?  And if not shouldn't it return an error considering its an IO based operation that could encounter an error?

Reply:
Good catch, this was following the original code. Will update to capture that error, and add a test case for it.

Reply:
Found the same thing in checkpoint file write method ~ `persist/fs/write.go`. Fixed these two, opened #346 as a followup.